src/Models/Binding.php

```diff
@@ -18,6 +18,9 @@ class Binding extends TerminusModel
      */
     public function getUsername()
     {
-        return $this->has('legacy_username') ? $this->get('legacy_username') : $this->get('username');
+        $database_runtime = $this->get('database_runtime');
+        return $this->has('legacy_username')
+            ? $this->get('legacy_username')
+            : $this->$database_runtime->get('username');
```

Comment:
looks like this should be `$this->get('database_runtime')->get('username')`

Reply:
It should just be this `$database_runtime->get('username');`

Reply:
Seems like there is assignment already to $database_runtime and this can be merged without assignment.